_includes/doc/admin-guide/options/value-pairs.md

```diff
@@ -13,7 +13,7 @@ formatted to JSON format, or directly used in a mongodb() destination.
 The following example selects every available information about the log
 message, except for the date-related macros (R\_\* and S\_\*), selects
 the .SDATA.meta.sequenceId macro, and defines a new value-pair called
-MSGHDR that contains the program name and PID of the application that
+MSGHDR that contains the program name and `PID` of the application that
```

Comment:
why is MSGHDR all caps, is a a macro or a field in this conect, because of `PID` shouldn't it be `MSGHDR`?